.travis.yml

```diff
@@ -197,7 +198,8 @@ matrix:
         - . ./ci/travis/ci.sh build
       script:
         - . ./ci/travis/ci.sh test_wheels
-        - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then . $TRAVIS_BUILD_DIR/ci/travis/build-docker-images.sh; fi
+        - export PATH="$HOME/miniconda3/bin:$PATH"
+        - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then python -m pip install docker && python $TRAVIS_BUILD_DIR/ci/travis/build-docker-images.py; fi
```

Comment:
can we install the docker as an extra step a step above (we can just install it unconditionally)?

Reply:
SGTM